### DISCUSSION TITLE
What is Logseq Searing API ?

### DISCUSSION THREAD

**Discussion by happy15** [EINDEX/logseq-copilot#39] [Q&A] [answered]
First, great work and thanks!

You mentioned "Logseq Searing API" and I googled but found nothing. 

So I just wondering what Logseq Searing API is exactly?

**Comment by EINDEX** [comment, marked as answer]
Sorry that is the typo of search, this is the search API I am using currently.

https://github.com/logseq/logseq/blob/7de92849dd8ab98f3feb220b51206f55d32a8125/src/main/logseq/api.cljs#L984-L988

And the other 2 APIs I am using:
q: https://github.com/logseq/logseq/blob/7de92849dd8ab98f3feb220b51206f55d32a8125/src/main/logseq/api.cljs#L836-L842
get_current_graph: https://github.com/logseq/logseq/blob/7de92849dd8ab98f3feb220b51206f55d32a8125/src/main/logseq/api.cljs#L145-L151
